docs/src/network_analysis/odes.md

```diff
@@ -0,0 +1,352 @@
+# [Decomposing the Reaction Network ODEs](@id network_analysis_odes)
```

Comment:
Maybe we should add a link to the intro page that Sam wrote describing how these ODEs are generated?